src/TodoList.js

```diff
@@ -9,7 +9,11 @@ function TodoList(props) {
         <ul>
             {props.todoList.map(({id,title})=> {
                 return(
-                <TodoListItem key={id} title={title}/>
+                <TodoListItem 
+                key={id} 
+                title={title}
+                onRemoveTodo = {() => props.onRemoveTodo(id)}
```

Comment:
let's remove the spacing in `onRemoveTodo`:

`onRemoveTodo={() => props.onRemoveTodo(id)}`

Reply:
Could you clarify this? Are you talking about the spacing around the => ? Thanks

Reply:
Yes, around the first = and around the =>. You already did it as such, that's great.

Reply:
Just around the first `=`. Whatever is inside the curly brackets can have regular spacing.

like this: onRemoveTodo={() => props.onRemoveTodo(id)}